src/Microsoft.AspNetCore.Razor.Runtime/TagHelpers/DefaultTagHelperContent.cs

```diff
@@ -342,10 +344,13 @@ public override Encoding Encoding
             public bool IsEmptyOrWhiteSpace { get; private set; } = true;
 
 #if NETSTANDARD1_5
```

Comment:
Nuke so it works the same on desktop and then add a test to ensure it properly picks up `!IsEmptyOrWhiteSpace`